governance.md

```diff
@@ -84,7 +84,16 @@ The initial composition of the SC is self-defined and is made of 10 members:
 
 #### Selection
 
-The initial SC members will be responsible for defining how and when the SC membership can be expanded or renewed.
+SC membership can be expanded by a unanimous vote of the commitee.
+Individuals wishing to join the SC should make a PR to this file updating the above list.
+The PR message should include a brief background of the proposed member.
+There are no eligibility requirements for membership.
+The SC committee will mainly consider community contributions and influence when voting on new members.
+This could include code contributions to the project, maintainership of project components, and participation in community meetings among other things.
```

Comment:
I think we value code, docs, issues, and other artifacts that get committed to repos. Likewise maintainership implies substantive reviews of those artifacts (i.e. PR reviews). 
I'm not sure about participation in meetings though. The things that come to mind for me are things that lead to the aforementioned artifacts.